client/my-sites/themes/style.scss

```diff
@@ -70,10 +70,12 @@
 	0 2px 4px lighten( $gray, 20% );
 }
 
-.pulsing-dot {
-	right: 24px;
-	top: 22px;
-	left: initial;
+.web-preview {
+	.pulsing-dot {
```

Comment:
Yeah I was about to make the same comment.
I checked and the `WebPreview` is attached higher in the hierarchy, so it's not inside themes exclusively.

It's probably worth moving to the WebPreview CSS. :)